packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -297,11 +299,26 @@ class ReorderableListView extends StatefulWidget {
   /// {@macro flutter.widgets.reorderable_list.dragBoundaryProvider}
   final ReorderDragBoundaryProvider? dragBoundaryProvider;
 
+  /// The cursor for a mouse pointer when it enters or is hovering over the drag
+  /// handle.
+  ///
+  /// If [mouseCursor] is a [WidgetStateMouseCursor],
+  /// [WidgetStateProperty.resolve] is used for the following [WidgetState]s:
+  ///
+  ///  * [WidgetState.hovered].
```

Comment:
Hovered should not be needed since cursors don't make sense unless hovered :)

Reply:
Makes sense, I've simplified it in [refactor: Simplify code](https://github.com/flutter/flutter/pull/160246/commits/ce1f0aab1f79608d7f84c4fd4cd78a0662ba1b90)